cli/mender-artifact/main.go

```diff
@@ -89,7 +80,7 @@ func getCliContext() *cli.App {
 
 	privateKeyFlag := cli.StringFlag{
 		Name: "key, k",
-		Usage: "Full path to the private key that will be used to verify " +
+		Usage: "Full path to the private key that will be used to sign " +
```

Comment:
Suggestion: Signing the artifact signature sounds a little weird. Reword into something like, `sign the Artifact`. Where Artifact should be capitalised of course. Btw, github is redlining aussie spelling for capitalised. Fuckers

Reply:
Haha!

Here I was thinking I was cleaning up, and I messed up the sentence! Indeed, "sign the artifact" is what it should be.